adapters/has_component_adapter.py

```diff
@@ -0,0 +1,93 @@
+from adapters.compound_adapter import CompoundAdapter
+import rdflib
+from owlready2 import *
+from tqdm import tqdm
+from adapters import Adapter
+from biocypher._logger import logger
+from adapters import *
+
+
+class HasComponentEdge(CompoundAdapter):
```

Comment:
This should be inheriting from Adapter not CompoundAdapter. We are trying to build independent adapters. I understand you wanted to reuse as much code as possible but this will create an unnecessary dependency between the Adapters. What do you think?

Reply:
> This should be inheriting from Adapter not CompoundAdapter. We are trying to build independent adapters. I understand you wanted to reuse as much code as possible but this will create an unnecessary dependency between the Adapters. What do you think?

yea you are right. i thought we can build on top of the compound adapter taking the compound adapter as a general adapter but as you mentioned there will be too much conflict that way. So, i agree with writing an independent adapters.